core/constants.py

```diff
@@ -2,3 +2,5 @@
 
 YETI_ROOT = path.normpath(path.join(path.dirname(path.abspath(__file__)), ".."))
 STORAGE_ROOT = path.join(YETI_ROOT, "storage")
+DB_VERSION = 0
```

Comment:
Maybe this is not clear, but the DB_VERSION constant is supposed to represent the database version the code is meant to work with. The "highest migration" is the highest number between DB_VERSION and whichever version is stored in the database. 

In the case we'd want to revert migrations, there should be a second function in each migration to revert this. Do we want this now?